src/components/GenreSelect.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+class GenreSelect extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      genre: "",
+    };
+
+    this.selectGenre = this.selectGenre.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  selectGenre(event) {
+    this.props.onSelect(event.target.value);
+  }
+
+  handleChange(event) {
+    this.setState({ genre: event.target.value });
+  }
+
+  render() {
+    return (
+      <div>
+        <p className="genreSelect-title">GENRE</p>
+        <select
```

Comment:
Please, update the component according to the design in Figma.
![image](https://user-images.githubusercontent.com/47754466/226386641-faea97c3-43ca-4ca6-9ec2-df376365212d.png)